README.md

```
@@ -28,6 +39,16 @@ Production-ready AWS infrastructure for Laravel applications using Terraform. Th
 - **Bastion Host** - Secure database access (optional)
 - **CloudTrail** - API audit logging (optional)
 - **Read Replicas** - Database read replicas for analytics (optional)
+- **Scheduled Scaling** - Automatic scale down during nights and weekends for cost savings (optional)
+
+### Compliance & Security Features
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)